<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMode</class>
 <widget class="QWidget" name="AdminMode">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminMode</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Buttons">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QPushButton" name="AddButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DeleteButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="UpdateButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ListButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>List</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="GraphButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Show Graph</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="UndoButton">
        <property name="text">
         <string>Undo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RedoButton">
        <property name="text">
         <string>Redo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
